back/test/data/size.py

```diff
@@ -9,7 +9,7 @@ def default_data():
 
 
 def another_data():
-    return {"id": 2, "label": "medium"}
+    return {"id": 2, "label": "medium", "size_range": size_range_data()["id"]}
```

Comment:
This addition broke the `test_product` test because the query there requests `sizeRange { sizes { id } }`. Hence there the expectation needs to be adjusted (add `another_size`)